### PR TITLE
Update to GOV.UK Frontend v3

### PR DESCRIPTION
**Pull request by chao-xian**
This updates the Service Manual to use the latest version of GOV.UK Publishing Components, which means version 3 🐝 of GOV.UK Frontend. 

Lots of bespoke markup and CSS deleted and replaced with components:
* Search component in hero on homepage
* Accordion component on topic template
* Feedback component on every page
* Header component on most pages

Trello card: https://trello.com/c/zArhsTSW/56-3-update-service-manual-frontend-to-govukpublishingcomponents-v19-with-compatibility-mode-on

Quick checks (locally, as no preview app):

* http://localhost:3122/service-manual/
* http://localhost:3122/service-manual/helping-people-to-use-your-service
* http://localhost:3122/service-manual/design
* http://localhost:3122/service-manual/service-assessments
* http://localhost:3122/service-manual/service-standard
* http://localhost:3122/service-manual/service-standard/point-1-understand-user-needs
* http://localhost:3122/service-manual/communities
* http://localhost:3122/service-manual/communities/accessibility-community

### CODE REVIEW

File: app/assets/stylesheets/modules/_hero.scss [hunk 1]
```diff
@@ -1,57 +1,26 @@
...
-    @include media(tablet) {
-      padding: ($gutter + $border-top-width) 0 $gutter;
-    }
-  }
-
-  &__title {
-    @include core-48;
-    margin-bottom: $gutter;
+    color: govuk-colour("white");
     font-weight: bold;
```
Comment (kr8n3r) [inline app/assets/stylesheets/modules/_hero.scss]: 700 or @include govuk-typography-weight-bold

Comment (injms) [reply app/assets/stylesheets/modules/_hero.scss]: Fix now - ta!

File: app/assets/stylesheets/modules/_service-standard-point.scss [hunk 2]
```diff
@@ -1,45 +1,41 @@
...
-    @include media(tablet) {
+    @include govuk-media-query($from: tablet) {
       margin-left: $gutter-tablet;
     }
   }
 
   &__number {
     float: left;
-    margin-left: -$gutter;
+    margin-left: (0 - govuk-spacing(6));
```
Comment (kr8n3r) [inline app/assets/stylesheets/modules/_service-standard-point.scss]: or just -(govuk-spacing(6))

File: app/assets/stylesheets/modules/_service-standard-point.scss [hunk 3]
```diff
@@ -1,45 +1,41 @@
...
 
-    @include media(tablet) {
+    @include govuk-media-query($from: tablet) {
       margin-left: -$gutter-tablet;
```
Comment (kr8n3r) [inline app/assets/stylesheets/modules/_service-standard-point.scss]: will it still exist with frontend_toolkit removal?

Comment (injms) [reply app/assets/stylesheets/modules/_service-standard-point.scss]: Yes, as `$gutter-tablet` is set within the [`_service-standard-point.scss` file](https://github.com/alphagov/service-manual-frontend/pull/518/files/875fb6963486a92e1d2ea6fcacbb4990c3d1a00c#diff-a2d4e3b902bbce25f8bb079c6be9473bR2) which is :

```
$gutter-tablet: (govuk-spacing(6) * 1.5);  // 45px
```

File: app/assets/stylesheets/modules/_service-standard-point.scss (hunk 2 above)
Comment (injms) [reply app/assets/stylesheets/modules/_service-standard-point.scss]: Ooh, neat. I thought that Sass required you to subtract from 0. I'll change this.

**Comment by kr8n3r** [comment]
there's some toolkit specific mixins left there. i guess the intention is to address that later?

**Comment by injms** [comment]
> there's some toolkit specific mixins left there. i guess the intention is to address that later?

Yeah, that's right - we can remove toolkit later. I've replaced some mixins as I've come across them in this upgrade, so there's a mix of Frontend Toolkit and GOV.UK Frontend mixins in this pull request.

File: app/views/content_items/topic.html.erb [hunk 4]
```diff
@@ -4,82 +4,73 @@
...
-        </h2>
-        <p class="related-item__description">
-          Find out what the cross-government community does and how to get involved.
-        </p>
-        <nav role="navigation" aria-labelledby="related-communities" class="related-communities">
-          <ul class="related-item__list">
-            <% @content_item.content_owners.each do |content_owner| %>
-              <li class="related-item__list-item">
-                <%= link_to content_owner.title, content_owner.href %>
+          <ul class="govuk-list subsection__list govuk-!-margin-bottom-9">
```
Comment (kr8n3r) [inline app/views/content_items/topic.html.erb]: looks like subsection list class was used to add more padding and remove text-underline. as we've removed that styling we could remove this class.

**Comment by kr8n3r** [comment]
there are some minor spacing differences but that's to be expected. 
Otherwise it looks good to go.

File: app/views/content_items/topic.html.erb (hunk 4 above)
Comment (injms) [reply app/views/content_items/topic.html.erb]: 👍

**Comment by chao-xian** [comment]
Thanks for your work on this @injms!

**Pull request merged**